packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -927,7 +928,7 @@ export const Table = <
             </div>
           )}
           {!isLoading &&
-            (data.length === 0 || filteredFlatRows.length === 0) &&
+            (data.length === 0 || rows.length === 0) &&
```

Comment:
Why did you change from `filteredFlatRows` to `rows` here?

Reply:
If I'm not mistaken, `filteredFlatRows` is rows after column filters are applied and `rows` are final rows after all the filters are applied